keyboards/ashpil/modelm_usbc/info.json

```
@@ -0,0 +1,15 @@
+        }
+		"LAYOUT_fullsize_ansi_wkl ": {
```

Another extra space.

```suggestion
		"LAYOUT_fullsize_ansi_wkl": {
```